server/booking.js

```diff
@@ -0,0 +1,80 @@
+import { Router } from "express";
+import { google } from "googleapis";
+
+import logger from "./utils/logger";
+import dayjs from "dayjs";
+
+const router = Router();
+
+const oauth2Client = new google.auth.OAuth2(
+	process.env.CLIENT_ID,
+	process.env.CLIENT_SECRET,
+	process.env.REDIRECT_URL
+);
+
+// Extract google calender api
+const calendar = google.calendar({
+	version: "v3",
+	auth: process.env.AUTH_KEY,
+});
+
+router.get("/", async (req, res) => {
+	logger.debug("Book a screening");
+
+	// Required
+	const {
+		name,
+		surname,
+		phoneNumber,
+		addressLine1,
+		addressLine2,
+		city,
+		country,
+		dateTime,
+	} = req.body;
+	const required = {
+		name,
+		surname,
+		phoneNumber,
+		addressLine1,
+		addressLine2,
+		city,
+		country,
+		dateTime,
+	};
+	// Optional
+	const { email, postCode } = req.body;
+
+	if (!Object.values(required).every((v) => v)) {
+		return res.status(500).json({
+			msg: "Could not make a booking, fill all required fields",
```

Comment:
prefer we use the word `message` here

Reply:
Would also use the correct status codes for errors, 500 is mainly reserved for internal server errors and might cause confusion

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status